README.md

```diff
@@ -149,6 +149,7 @@ Name | Website | Region
 [EngineYard (Support Team)](/company-profiles/engineyard.md) ⚠️️ | https://www.engineyard.com/ |
 [Enjoei](/company-profiles/enjoei.md) ⚠️️ | https://www.enjoei.com.br/ | Brazil
 [Envato](/company-profiles/envato.md) | https://envato.com/ | Worldwide
+[Epam](/company-profiles/epam.md) | https://epam.com/ | Belarus, Russia and Ukraine. 
```

Comment:
Region is shown as "Belarus, Russia and Ukraine." but the site linked in the company profile mentioned "Anywhere"

https://anywhere.epam.com/

If so, please change this to `Worldwide`

Reply:
ok, makes sense to me, thank you